modules/restarting-installation.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// installing/installing-troubleshooting.adoc
+
+[id="restarting-installation_{context}"]
+= Reinstalling the {product-title} cluster
+
+If you are unable to debug and resolve issues in the failed {product-title} installation, you might need to consider installing a new {product-title} cluster. Before starting the installation process again, you must do a thorough cleanup.
+For the user-provisioned infrastructure (UPI) installation, you must manually destroy the cluster and delete all the associated resources. For the installer-provisioned infrastructure (IPI) installation, see the following procedure.
+
+.Procedure
+
+1. Destroy the cluster and remove all the resources associated with the cluster, including the hidden installer state files in the installation directory:
```

Comment:
```suggestion
. Destroy the cluster and remove all the resources associated with the cluster, including the hidden installer state files in the installation directory:
```

Remove the hard-coded "1." and use "." instead. This applies to all steps below.

Reply:
Addressed.